test/configCases/errors/multi-entry-missing-module/index.js

```diff
@@ -0,0 +1,9 @@
+it("Should use WebpackMissingModule when module is missing with multiple entry setup", function() {
+  var fs = require("fs");
+  var source = fs.readFileSync("test/js/config/errors/multi-entry-missing-module/b.js", "utf-8");
```

Comment:
@sokra any idea why both 
```
fs.readFileSync("./b.js", "utf-8");
```
and 
```
fs.readFileSync(path.join(__dirname, "b.js"), "utf-8");
```

keep failing with file not found?

Reply:
You need to add

``` js
node: {
  __dirname: false
}
```

to config. This allows you to use node.js' `__dirname`